app/Nova/User.php

```diff
@@ -517,4 +524,75 @@ public function actions(Request $request): array
                 }),
         ];
     }
+
+    /**
+     * Get the search result subtitle for the resource.
+     */
+    public function subtitle(): ?string
+    {
+        $managed_team_names = $this->manages()->pluck('name');
+
+        if (count($managed_team_names) > 0) {
+            $team_name = $managed_team_names[0];
+
+            if ('Core' === $team_name) {
+                return 'President';
+            }
+
+            if ('Alumni Leadership' === $team_name) {
+                return 'Corporation President';
+            }
+
+            if (
+                str_contains($team_name, 'Training') ||
+                str_contains($team_name, 'Core') ||
+                str_contains($team_name, 'Outreach')
+            ) {
+                return $team_name.' Chair';
+            }
+
+            return $team_name.' Project Manager';
+        }
+
+        // This query is adapted from the dashboard controller
+        $paidTransactions = AppModelsDuesTransaction::select(
+            'dues_transactions.id',
+            'dues_transactions.dues_package_id',
+            'dues_packages.effective_start',
+            'dues_packages.effective_end',
+        )
+        ->leftJoin('payments', static function (JoinClause $join): void {
+            $join->on('dues_transactions.id', '=', 'payable_id')
+                 ->where('payments.payable_type', AppModelsDuesTransaction::getMorphClassStatic())
+                 ->where('payments.amount', '>', 0);
+        })
+        ->leftJoin('dues_packages', 'dues_transactions.dues_package_id', '=', 'dues_packages.id')
+        ->where('user_id', $this->id)
+        ->whereNotNull('payments.id')
+        ->orderBy('dues_packages.effective_start')
+        ->orderBy('dues_packages.effective_end')
+        ->get();
+
+        $firstPaidTransact = $paidTransactions->first();
+        $lastPaidTransact = $paidTransactions->last();
+
+        if (null !== $firstPaidTransact && null !== $lastPaidTransact) {
+            // @phpstan-ignore-next-line
+            $firstYear = (new Carbon($firstPaidTransact->effective_start, 'America/New_York'))->year;
+            // @phpstan-ignore-next-line
+            $lastYear = (new Carbon($lastPaidTransact->effective_end, 'America/New_York'))->year;
+
+            return $firstYear === $lastYear ? 'Member '.$firstYear : 'Member '.$firstYear.'-'.$lastYear;
+        }
+
+        $major_names = $this->majors()->pluck('display_name')->toArray();
+
+        $class_standing_names = $this->classStanding()->pluck('name')->toArray();
+
+        if (count($major_names) > 0 && null !== $major_names[0] && count($class_standing_names) > 0) {
+            return $major_names[0].' | '.ucfirst($class_standing_names[0]);
```

Comment:
Remove these pls

Reply:
Not necessarily opposed, but why? Is there a different value you'd like for non-members?

Reply:
Gah, GitHub mobile botched my comment location.  I meant the 2 `ray` lines above this

Reply:
Fixed in c0ba56ec9a5b783245f8ee76f44098f786e01d2a